libs/langchain/langchain/llms/bedrock.py

```diff
@@ -139,6 +153,36 @@ def prepare_output_stream(
                     text=chunk_obj[cls.provider_to_output_key_map[provider]]
                 )
 
+    @classmethod
+    async def aprepare_output_stream(
+        cls, provider: str, response: Any, stop: Optional[List[str]] = None
+    ) -> AsyncIterator[GenerationChunk]:
+        stream = response.get("body")
+
+        if not stream:
+            return
+
+        if provider not in cls.provider_to_output_key_map:
+            raise ValueError(
+                f"Unknown streaming response output key for provider: {provider}"
+            )
+
+        async for event in stream:
```

Comment:
Got an error using async
`TypeError: 'async for' requires an object with __aiter__ method, got EventStream`
Had to remove the async

Reply:
I just tested both sync and async ways of calling bedrock. Including with and without chains and I can't seem to re-create your issue. Would you mind sending over the code you used to get this?

Reply:
async for' requires an object with __aiter__ method, got EventStream same

Reply:
work around by remove "async" keyword